### PR TITLE
SystemInfo request introduced

### PR DESCRIPTION
**Pull request by cybay**

### CODE REVIEW

File: src/requests/system_info.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
+
+#include "requests/system_info.h"
+
+#include "context.h"
+#include "connection.h"
+#include "./../system_info.h"   // andrew.kibirev: yes, know that it's UGLY and Wrong. It'll be fixed.
```
Comment (bitkis) [inline src/requests/system_info.cpp]: time to fix has come? :)

File: src/requests/system_info.h [hunk 2]
```diff
@@ -0,0 +1,103 @@
+#pragma once
```
Comment (bitkis) [inline src/requests/system_info.h]: looks like a shared header

File: src/server.cpp [hunk 3]
```diff
@@ -91,16 +119,23 @@ void GraftServer::setCoapRouters(CoapConnectionManager& coapcm)
 
 void GraftServer::initGlobalContext()
 {
-//  TODO: why context intialized second time here?
-    graft::Context ctx(m_looper->getGcm());
-    const ConfigOpts& copts = m_looper->getCopts();
+//  //TODO: why context intialized second time here?
```
Comment (bitkis) [inline src/server.cpp]: clean up  in order

File: test/request/system_info.cpp [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+    //void override_config_values(pt::ptree& ini_data, po::variables_map& cmdline_data) override;
+};
+
+//void GraftServerForTest::override_config_values(pt::ptree& ini_data, po::variables_map& cmdline_data)
+//{
+//  std::cout << "GraftServerForTest::override_config_values called" << std::endl;
+//  ini_data.put("", 0);
+//}
+
+TEST(Supernode, DISABLED_RequestSystemInfo)
```
Comment (bitkis) [inline test/request/system_info.cpp]: so, do we  actually test?

File: src/requests/system_info.h (hunk 2 above)
Comment (cybay) [reply src/requests/system_info.h]: What is shared header? And what exactly does make it looking like this?
If you mean its location in src dir? 
Frankly I don't see why it should be located somewhere else.

File: src/server.cpp (hunk 3 above)
Comment (cybay) [reply src/server.cpp]: What exactly should be done here?
Revert this code to original state? As it was before this commit?
Or remove these commented and useless lines at all?

File: test/request/system_info.cpp (hunk 4 above)
Comment (cybay) [reply test/request/system_info.cpp]: As for testing. Probably I got it wrong but it seemed to me that we decided make it at first without test. And than make tests for it.

**Comment by bitkis** [comment]
should we close this?

**Pull request closed without merging**